action.yml

```diff
@@ -0,0 +1,27 @@
+name: "Generate A Python Client Package From An OpenAPI file"
```

Comment:
Is this the name that gets displayed in the marketplace? Or just in Actions when running?

If the first, this should be "openapi-python-client Action" or something. If the second, maybe just "Generate Python Client"?

Reply:
The former; My thinking was to make it a tad more search friendly (on the actions marketplace that is) so that its a bit discoverable + easy to get what it does just from reading the name. That seems to be the standard way of doing it based off of what I've seen in the marketplace - unless its a multipurpose/function action the name breifly tells you what it does. I think it could do with being shortened however - how do you feel about just "Generate Python Client From An OpenAPI File"?

Reply:
Or "Generate Python Client from OpenAPI"?